### PR TITLE
WEBAPP-329: Adjust feedback box

### PR DESCRIPTION
**Pull request by michael-markl**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/modules/feedback/components/FeedbackBox.js [hunk 1]
```diff
@@ -37,6 +37,12 @@ export const SubmitButton = styled.span`
   cursor: pointer;
 `
 
+const ModifiedDropDown = props => <Dropdown {...props} controlClassName={props.className} />
+
+const StyledDropdown = styled(ModifiedDropDown)`
+  cursor: pointer !important;
```
Comment (maxammann) [inline src/modules/feedback/components/FeedbackBox.js]: Is there still a pointer when you hover the dropdown? The problem was that the pointer would not disappear.

Comment (Schedulaar) [reply src/modules/feedback/components/FeedbackBox.js]: There's a pointer now when you hover over the drop-down.
Not sure what we wanted it to behave. It sounds like a bug of react-dropdown what you're saying. Wouldn't fix that but I'll need to have another look.

Comment (maxammann) [reply src/modules/feedback/components/FeedbackBox.js]: Maybe react-select is also a look worth: https://www.npmjs.com/package/react-select

But it has the same pointer behavior apparently, but is accessible

**Comment by maxammann** [comment]
Can this get merged?

**Comment by Schedulaar** [comment]
@maxammann Since the checks don't run: No... 🙄

**Comment by klinzo** [comment]
I think you could merge this now.

**Pull request merged**